php-ts-mode.el

```diff
@@ -283,9 +278,8 @@ Return nil if there is no name or if NODE is not a defun node."
   (setq-local treesit-font-lock-settings php-ts-mode--font-lock-settings)
   (setq-local treesit-font-lock-feature-list
               '((comment definition preprocessor)
-                (constant keyword string type variables)
-                (annotation expression literal function)
-                (bracket delimiter operator)))
+                (keyword string type variables)
```

Comment:
We should think about value levels. I suggest we do it as a follow up when we stabilise types that we want to use. I put it in a checklist in https://github.com/emacs-php/php-ts-mode/issues/8